tests/contrib/django/test_django_snapshots.py

```diff
@@ -42,7 +38,7 @@ def daphne_client(snapshot):
         env=env,
     )
 
-    client = Client("http://localhost:%d" % SERVER_PORT)
+    client = Client("http://localhost:%s" % SERVER_PORT)
```

Comment:
why change from `%d` to `%s`?

Reply:
https://github.com/DataDog/dd-trace-py/pull/2956#discussion_r775644897